### DISCUSSION TITLE
Configuration of dde-control-center and ramdomico wallpaper

### DISCUSSION THREAD

**Discussion by talesam** [linuxdeepin/developer-center#7011] [Developer Discussion | 开发者讨论]
I am a BigLinux developer and we are creating a DDE version, but due to lack of documentation some things that are simple become complex.

- When I activate the default image app in the control center, in which file is it saved? I couldn't figure it out, the default image app isn't set, so I need to leave it set.

- By default the theme and wallpaper are automatic, I believe I managed to leave the theme set, but the wallpapers always change automatically, how can I disable this?

**Comment by BLumia** [comment]
cc @Decodetalkers

**Comment by Decodetalkers** [comment]
Emm, it saved in dconfig, and a json file under ~/. config/dde-appearance I think, there's a repo about theme, the files named as "theme.ini" record the wallpaper location , what I remember is like this
